climada/engine/uncertainty/unc_impact.py

```diff
@@ -255,3 +256,29 @@ def plot_rp_distribution(self):
         ax.set_yticklabels(df_values.columns)
 
         return fig, ax
+    
+    
+    def plot_sensitivity_map(self, exp):
```

Comment:
The latest commit add the possibility to plot a sensitivity map which shows at each exposure location what is the largest sensitivity index. 

Please note that this commit depends on the branch `feature/categorical_plot` which will be put to review soon.